app/models/student.rb

```diff
@@ -6,4 +6,8 @@ class Student < ApplicationRecord
   has_many :coordinators, through: :enrollments
 
   validates :first_name, presence: true, length: { minimum: 3 }
+
+  def name
```

Comment:
If you think OOD, then there should be a base class like `person` that they inherit these kinds of methods from. Could be a good other ticket for someone to pick up